.changeset/plenty-chefs-check.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vanilla-extract/esbuild-plugin': minor
```

Comment:
Just double checking—the esbuild plugin is currently v0.0.1, but this will push it to v0.1.0. Is that correct? Or should it be a patch to maintain the current versioning scheme?